setup-user.sh

```diff
@@ -19,9 +19,19 @@ echo "postgresql user: $POSTGRES_USER" > /tmp/PGPASSWORD.txt
 echo "postgresql password: $POSTGRES_PASS" >> /tmp/PGPASSWORD.txt
 
 # Check user already exists
+echo "Creating superuser $POSTGRES_USER"
 RESULT=`su - postgres -c "psql postgres -t -c \"SELECT 1 FROM pg_roles WHERE rolname = '$POSTGRES_USER'\""`
 COMMAND="ALTER"
 if [ -z "$RESULT" ]; then
 	COMMAND="CREATE"
 fi
-su - postgres -c "psql postgres -c \"$COMMAND USER $POSTGRES_USER WITH SUPERUSER ENCRYPTED PASSWORD '$POSTGRES_PASS';\""
+su - postgres -c "psql postgres -c \"$COMMAND ROLE $POSTGRES_USER WITH LOGIN SUPERUSER CREATEDB CREATEROLE INHERIT REPLICATION  ENCRYPTED PASSWORD '$POSTGRES_PASS';\""
```

Comment:
aren't some of these redundant? i.e a superuser can createdb, createrole etc. 

The replication user must not have other properties, just replication (not superuser)

Reply:
I forgot to revert it to the original one will change it to 
`su - postgres -c "psql postgres -c \"$COMMAND USER $POSTGRES_USER WITH SUPERUSER ENCRYPTED PASSWORD '$POSTGRES_PASS';\""`